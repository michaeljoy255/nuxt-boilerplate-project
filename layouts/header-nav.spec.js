import { mount, createLocalVue } from '@vue/test-utils'
import Vuetify from 'vuetify'
import HeaderNavLayout from './header-nav'

const localVue = createLocalVue()

let wrapper

beforeEach(() => {
  const vuetify = new Vuetify()
  wrapper = mount(HeaderNavLayout, {
    localVue,
    vuetify,
  })
})

afterEach(() => {
  wrapper.destroy()
})

describe('Layout > header-nav', () => {
  test('matches the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
