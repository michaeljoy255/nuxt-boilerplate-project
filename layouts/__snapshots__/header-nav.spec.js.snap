// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout > header-nav matches the snapshot 1`] = `
"<div data-app=\\"true\\" class=\\"v-application v-application--is-ltr theme--light\\" id=\\"app\\">
  <div class=\\"v-application--wrap\\">
    <header class=\\"v-sheet theme--dark v-toolbar v-app-bar v-app-bar--clipped v-app-bar--fixed secondary\\" style=\\"height: 64px; margin-top: 0px; transform: translateY(0px); left: 0px; right: 0px;\\">
      <div class=\\"v-toolbar__content\\" style=\\"height: 64px;\\"><button type=\\"button\\" class=\\"v-app-bar__nav-icon v-btn v-btn--flat v-btn--icon v-btn--round theme--dark v-size--default\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-menu theme--dark\\"></i></span></button></div>
    </header>
    <nav class=\\"v-navigation-drawer v-navigation-drawer--clipped v-navigation-drawer--close v-navigation-drawer--fixed v-navigation-drawer--is-mobile theme--light\\" style=\\"height: 100vh; top: 0px; transform: translateX(-100%); width: 256px;\\">
      <div class=\\"v-navigation-drawer__content\\">
        <div class=\\"v-list v-sheet theme--light\\"><a tabindex=\\"0\\" class=\\"v-list-item v-list-item--link theme--light\\">
            <div class=\\"v-list-item__action\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate material-icons theme--light\\">home</i></div>
            <div class=\\"v-list-item__content\\">
              Home
            </div>
          </a> <a tabindex=\\"0\\" class=\\"v-list-item v-list-item--link theme--light\\">
            <div class=\\"v-list-item__action\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate material-icons theme--light\\">dashboard</i></div>
            <div class=\\"v-list-item__content\\">
              Test
            </div>
          </a></div>
      </div>
      <div class=\\"v-navigation-drawer__border\\"></div>
    </nav>
    <main class=\\"v-main\\" style=\\"padding: 64px 0px 0px 0px;\\">
      <div class=\\"v-main__wrap\\">
        <div></div>
      </div>
    </main>
  </div>
</div>"
`;
